### ISSUE TITLE
Error handling: show friendly message if user does not have an exchange license 

### ISSUE THREAD

**Issue by okms** [Puzzlepart/did#547]
- [x] I have searched for existing issues to avoid creating duplicates

### Description
A user added to did may or may not have an active exchange license. This is something which will cause any requests to /calendar to return a `400 bad request`. 

This feature would be to catch this and make sure we show a friendly error message along the lines of 

"Please contact your administrator to ensure your account has the required licenses" 

Basically, just make sure did does not bomb...
